newIDE/app/src/UI/SemiControlledAutoComplete.js

```diff
@@ -93,97 +91,103 @@ export default class SemiControlledAutoComplete extends React.Component<
     }
   }
 
-  disableDivider = (option: Object): boolean => {
-    if (option.type === 'separator') {
-      return true;
-    } else {
-      return false;
-    }
+  isOptionDisabled = (option: Option): boolean => {
+    // Writing shorthand, ie, return option.type === 'separator' fails flow test.
+    // All the cases of type Option have to be explicitly handled.
```

Comment:
Thanks for the explanation 👍
I think we can remove the comment because it's not helping for the code understanding (but I'll remember this is necessary for the review) (for such things, you can also comment on the PR, like a reviewer, to tell "here be careful when reviewing, because blabla").

For someone in the future that might want to simplify the code, this person will get the Flow error and will understand why there is the extra if/else :)

Reply:
Speaking of else! Extra nitpicking: you don't need the else ;)